libs/core/langchain_core/indexing/api.py

```diff
@@ -24,12 +24,14 @@
     cast,
 )
 
-from langchain_community.document_loaders.base import BaseLoader
+from langchain_core.document_loaders.base import BaseLoader
 from langchain_core.documents import Document
+from langchain_core.indexing.base import RecordManager
 from langchain_core.pydantic_v1 import root_validator
 from langchain_core.vectorstores import VectorStore
 
-from langchain.indexes.base import NAMESPACE_UUID, RecordManager
+NAMESPACE_UUID = uuid.UUID(int=1984)
```

Comment:
i know this is just a migration but a comment of what this is doing would be great